modules/metastorage/src/testFixtures/java/org/apache/ignite/internal/metastorage/server/SimpleInMemoryKeyValueStorage.java

```diff
@@ -835,6 +851,29 @@ public void advanceSafeTime(HybridTimestamp newSafeTime) {
         }
     }
 
+    @Override
+    public void saveCompactionRevision(long revision) {
+        throw new UnsupportedOperationException();
```

Comment:
Who will call this method?

Reply:
Nobody I suppose, just like the snapshots in the storage.

Reply:
If it's only needed in the RocksDB-related implementation, I suggest calling it directly and not via interface method (and remove the method from the interface)

Reply:
It seems right to me that this should be in the interface. For example, there are methods `org.apache.ignite.internal.metastorage.server.SimpleInMemoryKeyValueStorage#snapshot` and `org.apache.ignite.internal.metastorage.server.SimpleInMemoryKeyValueStorage#snapshot` that also do not work for the test storage. If anything, we will fix it later.